src/main/java/seedu/address/logic/commands/AddPairCommand.java

```diff
@@ -49,8 +50,13 @@ public AddPairCommand(Nric elderlyNric, Nric volunteerNric) {
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
         try {
-            model.addPair(elderlyNric, volunteerNric);
-            return new CommandResult(String.format(MESSAGE_ADD_PAIR_SUCCESS, elderlyNric, volunteerNric));
+            boolean issueWarning = model.addPair(elderlyNric, volunteerNric);
+            if (!issueWarning) {
+                return new CommandResult(String.format(MESSAGE_ADD_PAIR_SUCCESS, elderlyNric, volunteerNric));
+            } else {
+                return new CommandResult(String.format(
+                        MESSAGE_ADD_PAIR_SUCCESS + "\n" + MESSAGE_WARNING_REGION, elderlyNric, volunteerNric));
+            }
```

Comment:
I'm on the fence about whether or not a warning message is required since there is no meaningful way to inform the end user that the message is a warning. My suggestion would be to add an attribute to `CommandResult` which informs the UI component whether the result is `SUCCESS, WARNING or ERROR`. However, I'm not sure if this will introduce any breaking changes, nor is it worth the effort. What do you guys think?